collectors/timex.plugin/plugin_timex.c

```diff
@@ -44,7 +44,7 @@ void *timex_main(void *ptr)
 
         struct timex timex_buf = {};
         int sync_state = 0;
-        sync_state = adjtimex(&timex_buf);
+        sync_state = ntp_adjtime  (&timex_buf);
```

Comment:
This additional space between function name and argument is not usually used in our codes, so I will suggest you to remove it.